import-service/src/helpers/http-response.ts

```diff
@@ -0,0 +1,24 @@
+import { headers } from "../constants";
+
+export const HttpResponse = {
```

Comment:
another option here is to use middleware. For more details see 'middyfy' in my PR